src/app/core/admin/admin-list-manager/admin-list-manager.component.html

```diff
@@ -35,6 +36,7 @@
           <div class="default-item flex-row gap-small align-center">
             <fa-icon icon="grip-vertical" class="drag-handle"></fa-icon>
             <app-entity-field-label
+              [customFields]="activeFields"
               [field]="getFieldId(item)"
```

Comment:
in PR https://github.com/Aam-Digital/ndb-core/pull/3135/files this was switched to `item`, why not here?